lib/recipients/recipient.ex

```diff
@@ -37,43 +38,41 @@ defmodule PINXS.Recipients.Recipient do
   @doc """
   Create a recipient
   """
-  @spec create(Recipient.t(), PINXS.t()) :: {:ok, Recipient.t()} | {:error, PINXS.Error.t()}
-  def create(%Recipient{bank_account: bank_account} = recipient, %PINXS{} = config) when not is_nil(bank_account) do
+  def create(%Recipient{bank_account: bank_account} = recipient, config)
```

Comment:
So going forward here, `config` will be one of either the old-style `PINXS` config instance _or_ a `Tesla.Client` instance, right?

Reply:
Yea, but only during the transition, in V3 I'll remove all the old refs.

With this change, anyone with the existing approach can get on to 2.1.0 and will get deprecation warnings, but everything will still work as is.